Riot/Modules/Room/RoomViewController.m

```diff
@@ -1486,6 +1514,138 @@ - (void)destroy
     [super destroy];
 }
 
+#pragma mark - Start DM
+
+/**
+ Check whether the current room is a direct chat left by the other member.
+ */
+- (void)isDirectChatLeftByTheOther:(void (^)(BOOL isEmptyDirect))onComplete
+{
+    // In the case of a direct chat, we check if the other member has left the room.
+    if (self.roomDataSource)
+    {
+        NSString *directUserId = self.roomDataSource.room.directUserId;
+        if (directUserId)
+        {
+            [self.roomDataSource.room members:^(MXRoomMembers *roomMembers) {
+                MXRoomMember *directUserMember = [roomMembers memberWithUserId:directUserId];
+                if (directUserMember)
+                {
+                    MXMembership directUserMembership = directUserMember.membership;
+                    if (directUserMembership != MXMembershipJoin && directUserMembership != MXMembershipInvite)
+                    {
+                        onComplete(YES);
+                    }
+                    else
+                    {
+                        onComplete(NO);
+                    }
+                }
+                else
+                {
+                    NSLog(@"[RoomViewController] isEmptyDirectChat: the direct user has disappeared");
```

Comment:
I think you should use MXLogDebug instead of NSLog. You added other occurrences of NSLog in this file too.